### ISSUE TITLE
Unexpected output when reactivating plugin

### ISSUE THREAD

**Issue by thomaspark** [thomaspark/snowball#66]
I get this error message when reactivating the plugin:

```
The plugin generated 420 characters of unexpected output during activation. If you notice “headers
already sent” messages, problems with syndication feeds or other issues, try deactivating or removing
this plugin.
```

Screenshot:
![screen shot 2015-06-11 at 8 17 29 pm](https://cloud.githubusercontent.com/assets/900407/8121245/eba0c324-1076-11e5-8f1b-8c79a2846394.png)

**Comment by brianlee389** [comment]
I found out this was a database script issue that happens in the installation.
The error was:
WordPress database error: [Duplicate key name 'ID']
ALTER TABLE wp_snowball_blocks ADD UNIQUE KEY ID (id)

WordPress database error: [Duplicate key name 'ID']
ALTER TABLE wp_snowball_articles ADD UNIQUE KEY ID (id)

I was using unique key improperly and I instead used Primary Key because ID seems sufficient enough to determine it's uniqueness.

I used this site to help me figure it out.
http://wordpress.stackexchange.com/questions/3533/the-plugin-generated-80-characters-of-unexpected-output
